_datascience/data_sharing.md

```diff
@@ -0,0 +1,96 @@
+---
+title: NIH Data Sharing
+primary_reviewers: vortexing
+---
+
+## New Data Sharing Policy
+> What you should know about the new 2023 NIH Data Management and Sharing Policy
+
+Starting January 25th, 2023, the NIH will implement a new policy that requires new grant applications or renewals that generate scientific data to include a detailed data management and sharing plan (DMSP). The NIH expects all shareable data to be made available, whether it is associated with a publication or not. The DMS Policy is similar to what other funding agencies call a data management plan (DMP). The goals of this new policy are to establish the expectation for the management and sharing of scientific data generated from NIH-funded or conducted research, and to emphasize the importance of good data management practices.
```

Comment:
Are these two phrases referencing the same thing?

> ... to include a detailed data management and sharing plan (DMSP) ...

and

> ... The DMS Policy is similar to what ...

Should the second one be "DMSP Policy"?  Or is "DMS Policy" a different plan/policy?

Reply:
Or are they both different?  A DMS Plan and the DMS Policy?  DMS Policy is referenced down doc, and "DMSP" is never used outside this introduction.  Maybe change the first reference to:

>    ... to include a detailed data management and sharing plan (DMS plan) ...

Reply:
Good catch.  Yeah, the DMS policy requires a DMS plan. ;) I'll disambiguate!!!